app/test_main.py

```diff
@@ -1 +1,58 @@
-# write your code here
+from unittest import mock
+from unittest.mock import MagicMock
+
+import pytest
+
+from app.main import can_access_google_page
+
+
+url = "https://google.com"
```

Comment:
It is better to use upper case here
```suggestion
TEST_URL = "https://google.com"
```